src/release.js

```diff
@@ -67,6 +76,12 @@ module.exports = function taoExtensionReleaseFactory(params = {}) {
     if (!adaptees[subjectType]) {
         throw new Error(`No implementation found for the type '${subjectType}'`);
     }
+
+    //in non TTY shells we turn off the interactive mode
+    if (!process.stdin.isTTY) {
```

Comment:
question: why did you decide to add this in addition to explicit `--no-interactive` option? I'm just a bit worried because 'non-interactive' can be thought of as a 'force' mode, since it doesn't ask for any confirmations.
If it's not possible for 'human' user to accidentally get into this mode and it is only for CI, or if it's a standard practice, then ok.

Reply:
Yes, I thought thoughtfully about this, and gather ppl opinions. 
The tool has multiple usages (or intended to have):  
 - _casual release_, a developer will release it's own extension or repo at some point. Most likely the interactive mode fits his need
 - _automated release_ (through ci), especially for an intermediary repository, the release will be done by the CI on PR merge, using non-TTY env (non interactive)
 - _manual batches of release_s, this is used to release cross company features, like the version removal. The release is done manually but on a bulk of repositories, and it can be handy to run in non-interactive mode (or from a bash script for example)

If someone runs it by mistake (like any other command), then we can still remove the release.

Reply:
Ok, thank you for explanation